waddrmgr/scoped_manager.go

```diff
@@ -1973,6 +1973,16 @@ func (s *ScopedKeyManager) importPublicKey(ns walletdb.ReadWriteBucket,
 		}
 		addressID = btcutil.Hash160(witnessScript)
 
+	case TaprootPubKey:
+		internalPubKey, err := btcec.ParsePubKey(serializedPubKey)
```

Comment:
This is because we're in legacy code here. We're either called from `ImportPrivateKey` or `ImportPublicKey` which both call `SerializeCompressed()`. Didn't want to add a lot of refactor code to this PR but can try to clean that up in a follow-up PR.